app/src/main/res/values/styles.xml

```diff
@@ -1,14 +1,24 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light">
```

Comment:
This shouldn't be changed, since we're gonna need the ActionBar later in application.